lib/src/repo.rs

```diff
@@ -1329,6 +1329,21 @@ impl MutableRepo {
         Ok(num_rebased)
     }
 
+    pub fn reparent_descendants(&mut self, settings: &UserSettings) -> BackendResult<usize> {
```

Comment:
Can you add a doc comment?

Sorry that I forgot to ask before, and sorry that `rebase_descendants()` (which was probably added by me) doesn't have it. I hope to some day find time to go through and add missing doc comments so we can remove `#![allow(missing_docs)]` everywhere.

Reply:
I've added an initial commit with documentation for `rebase_descendants()`, and added the documentation for `reparent_descendants()`.